### PR TITLE
Added photobooth to staging + remove from admin + fix display styles

### PR DESCRIPTION
**Pull request by alexey-petrov**
Closes https://github.com/sparkletown/internal-sparkle-issues/issues/1142
Implemented photobooth on staging + remove from admin + fix display styles

### CODE REVIEW

File: src/components/molecules/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -219,6 +229,15 @@ export const NavBar: React.FC<NavBarPropsType> = ({
               <VenuePartygoers />
             </div>
 
+            {withPhotobooth && (
+              <div
+                className="NavBar__photobooth-button nav-schedule"
```
Comment (yurii-lubynets) [inline src/components/molecules/NavBar/NavBar.tsx:234]:
```suggestion
                className="NavBar__photobooth--button nav-schedule"
```

File: src/components/molecules/NavBar/NavBar.tsx [hunk 2]
```diff
@@ -219,6 +229,15 @@ export const NavBar: React.FC<NavBarPropsType> = ({
...
+                onClick={handlePhotoboothRedirect}
+              >
+                <p className="NavBar__photobooth-title">Photobooth</p>
```
Comment (yurii-lubynets) [inline src/components/molecules/NavBar/NavBar.tsx:237]:
```suggestion
                <p className="NavBar__photobooth--title">Photobooth</p>
```

File: src/components/molecules/NavBar/NavBar.scss [hunk 3]
```diff
@@ -270,4 +272,30 @@ $border-radius: 28px;
   &__profile-popover {
     height: 100%;
   }
+
+  &__photobooth {
+    &-button {
```
Comment (yurii-lubynets) [inline src/components/molecules/NavBar/NavBar.scss:277]:
```suggestion
    &--button {
```

Comment (yurii-lubynets) [inline src/components/molecules/NavBar/NavBar.scss:277]:
```suggestion
    &--button {
```

File: src/components/molecules/NavBar/NavBar.scss [hunk 4]
```diff
@@ -270,4 +272,30 @@ $border-radius: 28px;
...
+      background-size: $photobooth-button-background-size;
+      background-position: 50% 10%;
+      background-repeat: no-repeat;
+
+      &:hover {
+        background-color: $light-grey;
+      }
+    }
+
+    &-title {
```
Comment (yurii-lubynets) [inline src/components/molecules/NavBar/NavBar.scss:295]:
```suggestion
    &--title {
```

**Review by yurii-lubynets — COMMENTED** [review]
LGTM. Some BEM-consistent comments left

File: src/components/molecules/NavBar/NavBar.scss (hunk 4 above)
Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.scss:295]: Title isn't modifier here. The & is making it a bit more complex to notice that it's actually part of a more complex name.

`NavBar__photobooth-title`

My suggestion would be to just unroll this case as `&__photobooth-title` without the double `&` concatenation.

File: src/components/molecules/NavBar/NavBar.scss (hunk 3 above)
Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.scss:277]: Title isn't modifier here. The & is making it a bit more complex to notice that it's actually part of a more complex name.

`NavBar__photobooth-button`

My suggestion would be to just unroll this case as `&__photobooth-button` without the double `&` concatenation.

File: src/components/molecules/NavBar/NavBar.tsx (hunk 1 above)
Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.tsx:234]: Same as https://github.com/sparkletown/sparkle/pull/2336#discussion_r714654480

File: src/components/molecules/NavBar/NavBar.tsx (hunk 2 above)
Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.tsx:237]: Same as https://github.com/sparkletown/sparkle/pull/2336#discussion_r714654480

File: src/components/molecules/NavBar/NavBar.scss (hunk 3 above)
Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.scss:277]: Same as https://github.com/sparkletown/sparkle/pull/2336#discussion_r714654480

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx [hunk 5]
```diff
@@ -112,7 +112,11 @@ export const AdminVenueView: React.FC = () => {
   }
 
   return (
-    <WithNavigationBar hasBackButton={false} withSchedule={false}>
+    <WithNavigationBar
+      hasBackButton={false}
+      withSchedule={false}
+      withPhotobooth={false}
```
Comment (goran-peoski-work) [inline src/components/organisms/AdminVenueView/AdminVenueView.tsx:118]: Will this work?
```suggestion
```

File: src/components/organisms/WithNavigationBar/WithNavigationBar.tsx [hunk 6]
```diff
@@ -24,11 +24,13 @@ const NavBar = lazy(() =>
 export interface WithNavigationBarProps {
   hasBackButton?: boolean;
   withSchedule?: boolean;
+  withPhotobooth?: boolean;
 }
 
 export const WithNavigationBar: React.FC<WithNavigationBarProps> = ({
   hasBackButton,
   withSchedule = true,
+  withPhotobooth = true,
```
Comment (goran-peoski-work) [inline src/components/organisms/WithNavigationBar/WithNavigationBar.tsx:33]: Might be better to always have these boolean props default to `false`, that way it is a simplified use:

- `<WithNavigationBar>` when none are available,
- `<WithNavigationBar hasBackButton hasSchedule hasPhotobooth >` when all are available

Comment (alexey-petrov) [reply src/components/organisms/WithNavigationBar/WithNavigationBar.tsx]: agree, redone this logic in b230401

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx (hunk 5 above)
Comment (alexey-petrov) [reply src/components/organisms/AdminVenueView/AdminVenueView.tsx]: yep, done b230401

File: src/components/molecules/NavBar/NavBar.scss (hunk 4 above)
Comment (alexey-petrov) [reply src/components/molecules/NavBar/NavBar.scss]: done b230401

File: src/components/molecules/NavBar/NavBar.scss (hunk 3 above)
Comment (alexey-petrov) [reply src/components/molecules/NavBar/NavBar.scss]: fixed b230401

Comment (alexey-petrov) [reply src/components/molecules/NavBar/NavBar.scss]: fixed b230401

File: src/components/molecules/NavBar/NavBar.tsx (hunk 1 above)
Comment (alexey-petrov) [reply src/components/molecules/NavBar/NavBar.tsx]: fixed b230401

File: src/components/molecules/NavBar/NavBar.tsx (hunk 2 above)
Comment (alexey-petrov) [reply src/components/molecules/NavBar/NavBar.tsx]: fixed b230401

File: src/components/molecules/NavBar/NavBar.scss (hunk 3 above)
Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.scss]: @alexey-petrov but why "fix" something that wasn't broken? :D @yurii-lubynets look what you did now :D

File: src/components/molecules/NavBar/NavBar.scss [hunk 7]
```diff
@@ -270,4 +272,30 @@ $border-radius: 28px;
   &__profile-popover {
     height: 100%;
   }
+
+  &__photobooth {
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.scss:276]: My suggestion, to nip these misunerstandings in the bud

```scss
    &__photobooth-button {
    }

    &__photobooth-title {
    }
```

Comment (alexey-petrov) [reply src/components/molecules/NavBar/NavBar.scss]: fixed in 0f31efb

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/pages/VenuePage/TemplateWrapper.tsx [hunk 8]
```diff
@@ -144,18 +143,20 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
...
-            <AnnouncementMessage isAnnouncementUserView />
-
-            <Suspense fallback={<LoadingPage />}>{template}</Suspense>
-
-            {shouldShowChat && <ChatSidebar venue={venue} />}
-          </WithNavigationBar>
-        </ReactionsProvider>
-      </RelatedVenuesProvider>
-    </WorldUsersProvider>
+    <RelatedVenuesProvider venueId={venue.id}>
```
Comment (mike-lvov) [inline src/pages/VenuePage/TemplateWrapper.tsx:146]: It should be removed because this Provider is inside the WIthNavigationBar

**Review by mike-lvov — CHANGES_REQUESTED** [review]

File: src/pages/VenuePage/TemplateWrapper.tsx (hunk 8 above)
Comment (alexey-petrov) [reply src/pages/VenuePage/TemplateWrapper.tsx]: done 7b5d338

**Comment by alexey-petrov** [comment]
@whoaitsaimz my bad, I've just created a PR to remove this from staging https://github.com/sparkletown/sparkle/pull/2392

**Pull request merged**